content/cloud-docs/api-docs/organizations.mdx

```diff
@@ -800,12 +800,14 @@ curl \
 }
 ```
 
-## Relationships
+## Available Related Resources
```

Comment:
I don't think this is true. The relationships exist, but [only the "entitlement_set" relationship can be included](https://github.com/hashicorp/atlas/blob/main/app%2Fcontrollers%2Fapi%2Fv2%2Forganizations_controller.rb#L155-L157).